src/EA.Weee.Domain/Evidence/EvidenceNoteFilter.cs

```diff
@@ -28,6 +28,8 @@ public class EvidenceNoteFilter
 
```

Comment:
Would be tempted to update the constructor of EvidenceNoteFilter to take a NoteType as we always require that.